### DISCUSSION TITLE
DBeaver plan to support the EnterpriseDB?

### DISCUSSION THREAD

**Discussion by leehj0727** [dbeaver/dbeaver#9052] [Ideas]
#### information: 
- Operating system (distribution) and version : Windows 10 1909(OS build 18363.900)
- dbeaver version : 7.1.0.202005311732
- Database name and version : EnterpriseDB 10.9.17

#### Describe your question:

Hello.

I had a lot of discomfort while using pgAdmin4, so I found dbeaver while looking for an alternative.

The dbeaver has confirmed that it supports many databases. When creating a connection. 

EDB was also listed, but JDBC download error occurred, so I try PostgreSQL JDBC and connected.

The SQL editor was perfect, but there was a serious problem:  the package groups do not exist and are marked as schemas.

I searched for the problem and found the following information.  EnterpriseDB has packages, PostgreSQL does not.

- [Please support EnterpriseDB (EDB) derivative of PostgreSQL #6118](https://github.com/dbeaver/dbeaver/issues/6118)

And I further checked the linked issue and tried but it didn't work. Still getting the same error
(jdbc ver : edb-jdbc-42.2.9.1-1)

- [EDB extension #6135](https://github.com/dbeaver/dbeaver/issues/6135#issuecomment-542896728)

After that issue, it seems that there has been no progress for a long time.

Does dbeaver have any plans to support EDB?

**Comment by uslss** [comment]
Hi @leehj0727 ,
unfortunately, we are waiting for any activities from the EDB side. If you really need it, you could write to EDB support.
Best regards,
